src/components/activities/ActivityListPage.tsx

```diff
@@ -25,7 +25,9 @@ function ActivityList({ activities }: { activities: Activity[] }) {
             <td style={{ paddingRight: '1em' }}>
               <Link href={getActivityPath(a)}>{a.title}</Link>
             </td>
-            <td>{formatDate(a.startTime, 'yyyy-MM-dd EEE HHmm')}</td>
+            <td>
+              <OutputTime label="" time={a.startTime}></OutputTime>
```

Comment:
Absolute seems like the right answer here since the primary use case is to look for events that have expired from the main page, i.e. more than 1 day ago in any case